helm/values.yaml

```diff
@@ -519,47 +503,7 @@ postanalytics:
     extraVolumeMounts: []
     # - name: something
     #   mountPath: /mnt/something
-  cron:
-    config:
-      ### Cron jobs configuration
-      ###
-      "export-attacks":
-        schedule: "* * * * *"
-        command: "/opt/wallarm/ruby/usr/share/wallarm-common/export-attacks -l STDOUT"
-        timeout: 3h
-      "export-counters":
-        schedule: "* * * * *"
-        command: "/opt/wallarm/ruby/usr/share/wallarm-common/export-counters -l STDOUT"
-        timeout: 11m
-      "export-environment":
-        schedule: "0 */1 * * *"
-        command: "/opt/wallarm/ruby/usr/share/wallarm-common/export-environment -l STDOUT"
-        timeout: 10m
-      "brute-detect":
-        schedule: "* * * * *"
-        command: "/opt/wallarm/ruby/usr/share/wallarm-common/brute-detect -l STDOUT"
-        timeout: 6m
-      "sync-markers":
-        schedule: "* * * * *"
-        command: "/opt/wallarm/ruby/usr/share/wallarm-common/sync-markers -l STDOUT"
-        timeout: 1h
-      "weak-jwt-detect":
-        schedule: "* * * * *"
-        command: "/opt/wallarm/ruby/usr/share/wallarm-common/weak-jwt-detect -l STDOUT"
-        timeout: 10m
-      "sync-node":
-        schedule: "*/2 * * * *"
-        command: "/opt/wallarm/ruby/usr/share/wallarm-common/syncnode -f -p -r 120 -l STDOUT -L DEBUG"
-      "export-blocked-stats":
-        schedule: "* * * * *"
-        command: "/opt/wallarm/ruby/usr/share/wallarm-common/export-blocked-stats -l STDOUT -L DEBUG"
-        timeout: 24h
-
-    image:
-      registry: docker.io
-      image: wallarm/ingress-ruby
-      tag: "4.8.1-1"
-      pullPolicy: IfNotPresent
+  supervisord:
```

Comment:
let's add logLevel for commands and supervisord like it's done Ingress repo
* https://github.com/wallarm/ingress/blob/main/charts/ingress-nginx/values.yaml#L998
* https://github.com/wallarm/ingress/blob/main/charts/ingress-nginx/values.yaml#L1027

Reply:
it is possible to add on the controller side without problem as you mentioned

but we should remember that the actual sidecar container (proxy) also runs scheduled scripts, and introducing per-script debug levels there would be... problematic, the only idea i have is to introduce a lot of env vars which we propagate through `files/template.yaml.tpl` , or maybe introduce only one var (and use it for all scripts in bulk)

anyway, what kind of design do we want?
if we only introduce in the controller containers and avoid sidecar proxy, this is sort of inconsistent IMO